akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/ParameterDirectivesSpec.scala

```diff
@@ -49,15 +49,6 @@ class ParameterDirectivesSpec extends FreeSpec with GenericRoutingSpec with Insi
           }
         }
       }
-      "cause a MalformedRequestContentRejection on invalid query strings" in {
```

Comment:
Mainly because this type of error doesn't exist anymore. I can adapt to another failing case.